data/migrations/versions/b1c2d3e4f5a6_add_org_mirror_repo_creation_failed_log.py

```
@@ -0,0 +1,30 @@
+Create Date: 2026-01-28 10:00:00.000000
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = "b1c2d3e4f5a6"
+down_revision = "b2c3d4e5f6a7"
+
+import sqlalchemy as sa
+
+
+def upgrade(op, tables, tester):
```

Ruff (0.14.14)

[warning] Unused function argument: `tester`

(ARG001)